src/index.html

```diff
@@ -1,14 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+    content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+    href="./style.css">
+    <link rel="preconnect"
+    href="https://fonts.googleapis.com/">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com/"
+    crossorigin="true">
```

Comment:
Need to fix code style everywhere
```suggestion
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com/"
      crossorigin="true"
    >
```